cpp/include/IcePatch2/ClientUtil.h

```diff
@@ -107,34 +107,28 @@ class ICEPATCH2_API Patcher
 };
 typedef std::shared_ptr<Patcher> PatcherPtr;
 
-//
-// IcePatch2 clients instantiate the IcePatch2::Patcher class
-// using the patcher factory.
-//
+// IcePatch2 clients instantiate the IcePatch2::Patcher class using the patcher factory.
 class ICEPATCH2_API PatcherFactory
 {
 public:
 
-    //
-    // Create a patcher using configuration properties. The following
-    // properties are used to configure the patcher:
-    //
-    // - IcePatch2.InstanceName
-    // - IcePatch2.Endpoints
-    // - IcePatch2.Directory
-    // - IcePatch2.Thorough
-    // - IcePatch2.ChunkSize
-    // - IcePatch2.Remove
-    //
-    // See the Ice manual for more information on these properties.
-    //
-    static PatcherPtr create(const Ice::CommunicatorPtr&, const PatcherFeedbackPtr&);
-
-    //
-    // Create a patcher with the given parameters. These parameters
-    // are equivalent to the configuration properties described above.
-    //
-    static PatcherPtr create(const FileServerPrxPtr&, const PatcherFeedbackPtr&, const std::string&, bool, std::int32_t, std::int32_t);
+    /// Create a patcher with the given parameters.
+    /// @server The proxy to the IcePath2 server.
+    /// @feedback The feedback object to use to report progress.
```

Comment:
I didn't realize Doxygen support this syntax. I think we usually do `@param`. Is this the preferred syntax?

Reply:
These should indeed be `@param <name>`.